tests/Tests/700__sandbox/700__sandbox.robot

```diff
@@ -0,0 +1,115 @@
+*** Settings ***
+Documentation    Test Suite for Sandbox Test cases
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Library         String
+Library         SeleniumLibrary
+Suite Setup     Begin Web Test
+Suite Teardown  End Web Test
+
+
+*** Variables ***
+${SANDBOX_AUTH_TYPE}   *******
+${SANDBOX_USERNAME}    *******
+${SANDBOX_PASSWORD}    *******
+${SANDBOX_OCP_URL}     *******
+
+
+*** Test Cases ***
+Verify Application Switcher Have Only Opneshift Console Link
+    [Documentation]    Application switcher should have only
+    ...    openshift conosle link
+    [Tags]     ODS-309
+    ...        Sandbox
+    Check Openshift Console
+
+Verify ISV integration enablement IS Disabled
+    [Documentation]    All the isv integration
+    ...   component should be diabled
+    [Tags]     ODS-530
+    ...        Sandbox
+    Click Link    Explore
+    Capture Page Screenshot   explor.png
+    Wait Until Cards Are Loaded
+    Sleep    5s
+    ${link_elements}  Get WebElements    //article[@class="pf-c-card odh-card m-disabled"]
+    ${length}      Get Length    ${link_elements}
+    Run Keyword If  ${length} != ${11}    Fail     '${length}' tiles in Explore section is disabled
+
+Verify Support Link Doesn't Exist
+    [Documentation]   Only documentaion link is present
+    [Tags]    ODS-526
+    ...       Sandbox
+    Click Element    xpath=//*[@id="toggle-id"]
+    ${links_avialble}   Get WebElements
+    ...    //a[@class="odh-dashboard__external-link pf-c-dropdown__menu-item" and not(starts-with(@href, '#'))]
+    FOR  ${link}  IN  @{links_avialble}
+         ${href}    Get Element Attribute    ${link}    href
+         Run Keyword And Continue On Failure   Should Not Contain   ${href}   support   ignore_case=True
+    END
```

Comment:
could you move the for loop out of the test case?
Based on the doc tag, maybe you should add the check that the documentation link is present

Reply:
i have move it to keyword